File: TestApp/TestApp/Model/Human/Person.m
#import "Person.h"

@interface Person ()

// Private interface goes here.

@end

@implementation Person

// Custom logic goes here.
- (NSString *)personDescription{

    return [NSString stringWithFormat:@"%@ %@, %@",self.fName, self.lName, self.age];
}
@end

File: TestApp/TestApp/Model/Human/Person.swift
//
//  Person.swift
//  TestApp
//
//  Created by Dirk Lewis on 5/2/15.
//  Copyright (c) 2015 VHH. All rights reserved.
//

import Foundation

@objc(Person)
class Person:_Person {
    
    func personDescription()->String{
        return "\(self.fName) \(self.lName), \(self.age)"
    }
}
